src/ClientBase.php

```diff
@@ -52,10 +111,8 @@ public function get($endpoint, array $headers = [])
             }
```

Comment:
I think I previously had a comment about allowing to override the debug option, now that we support passing options:

```
if (!isset($options['debug']) && $this->config['env'] === self::ENV_DEVELOPMENT) {
  $options['debug'] = true;
}
```

Using capitals for `true/false` btw is a Drupal convention, general in PHP libraries we should be using lowercase (that was my code actually).

Reply:
I see you have added the code to the `buildClient` function, it should be removed from here then.